src/client/components/SignInButton/Button.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './Button.css';
+
+const Button = ({ click }) => {
```

Comment:
It should be named `onClick` since all callbacks are prefixed with `on`.
Also, you need to add prop types.